components/common/AppButton.vue

```diff
@@ -4,8 +4,15 @@
   <button
     type="button"
     class="select-none call-to-action text-secondary font-medium py-2 px-6 shadow items-center justify-center"
+    @click="$emit('click', $event)"
   >
-    <img v-if="icon" :src="icon" alt="ocean_pearl" class="inline-block mr-2" width="20" />
+    <img
+      v-if="icon"
+      :src="icon"
+      alt="ocean_pearl"
```

Comment:
suggestion(blocking): remove "_" from alt texts and use more descriptive alt-texts